<launch>
  <node respawn="true" pkg="joy" type="joy_node" name="jessicar2_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="baud" value="115200"/>
  </node>

  <!-- keyboard teleoperation node for robot -->
  <arg name="set_multi_robot_name" default=""/>
  <node pkg="jessicar2_teleop" type="jessicar2_teleop_joy.py" name="jessicar2_teleop_joystick" output="screen">
    <param name="max_fwd_vel" value="0.5" />
    <param name="max_rev_vel" value="0.5" />
    <param name="max_ang_vel" value="0.5" />
    <param name="tf_prefix" value="$(arg set_multi_robot_name)"/>
  </node>
</launch>
